crates/aptos/src/stake/mod.rs

```diff
@@ -22,6 +22,9 @@ pub enum StakeTool {
     InitializeStakeOwner(InitializeStakeOwner),
     SetOperator(SetOperator),
     SetDelegatedVoter(SetDelegatedVoter),
+    UnlockVestingRewards(UnlockVestingRewards),
+    UnlockVestedCoins(UnlockVestedCoins),
+    VestingDistribute(VestingDistribute),
```

Comment:
These three things don't mean much to me, and as a user I wouldn't be able to tell between the three.

Reply:
Any better suggestions for naming then?